practico_01/ejercicio_12.py

```diff
@@ -43,7 +47,12 @@ def combinar_enumerate(nombres: List[str], precios: List[float], ids: List[int])
     """Re-Escribir utilizando enumerate y agregando un nuevo componente.
     Referencia: https://docs.python.org/3/library/functions.html#enumerate
     """
-    pass # Completar
+    pass  # Completar
+    union = [(p1, p2, p3) for in1, p1 in enumerate(nombres)
+             for in2, p2 in enumerate(precios)
+             for in3, p3 in enumerate(ids)
+             if in3 == in2 and in2 == in1]
+    return tuple(union)
```

Comment:
Incorrecto, se debe resolver SIN usar comprensiones y utilizando un UNICO bucle for. Leer documentación de enumerate